ngx-graph-frontend/src/app/core/services/graph.service.ts

```diff
@@ -20,9 +20,9 @@ export class GraphDTO {
       "nodes": nodes,
```

Comment:
Please create separate PR's when there is no semantic change.
The review is hard to follow like this.